src/controllers/project.js

```diff
@@ -0,0 +1,99 @@
+// Project Controller
+import * as projectService from '../services/project';
+import * as userProjectService from '../services/userProject';
+import jwt from 'jsonwebtoken';
```

Comment:
Import npm packages at top and then import custom code separated by a blank line
```js
import jwt from 'jsonwebtoken';

import * as projectService from '../services/project';
import * as userProjectService from '../services/userProject';
```